src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4902,6 +4908,53 @@ void ReportZoneMeanAirTemp(EnergyPlusData &state)
     // PURPOSE OF THIS SUBROUTINE:
     // This subroutine updates the report variables for the AirHeatBalance.
 
+    if (state.dataHeatBalAirMgr->CalcExtraReportVarMyOneTimeFlag) {
+        for (auto const *reqVar : state.dataOutputProcessor->reqVars) {
+            if (reqVar->name == "ZONE WETBULB GLOBE TEMPERATURE" || reqVar->name == "SPACE WETBULB GLOBE TEMPERATURE") {
+                if (reqVar->key.empty()) {
+                    for (int ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) {
+                        if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+                            for (int spaceNum : state.dataHeatBal->Zone(ZoneLoop).spaceIndexes) {
+                                auto &thisSpaceAirRpt = state.dataHeatBal->spaceAirRpt(spaceNum);
+                                thisSpaceAirRpt.ReportWBGT = true;
+                            }
+                        }
+                        auto &thisZnAirRpt = state.dataHeatBal->ZnAirRpt(ZoneLoop);
+                        thisZnAirRpt.ReportWBGT = true;
+                    }
```

Comment:
Sorry I've made zone vs space so confusing. This code here will set `ReportWBGT=true` for all zones and spaces if either "Zone Wetbulb..." or "Space Webulb..." is requested. The structure should be more like:
```
if (reqVar->name == "ZONE WETBULB GLOBE TEMPERATURE"){
  if (reqVar->key.empty()) {
      set true for all zones
  else
      set for matching zone name.
}
}
if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
if (reqVar->name == "SPACE WETBULB GLOBE TEMPERATURE"){
  if (reqVar->key.empty()) {
      set true for all spaces
  else
      set for matching space name.
}
}
}

Reply:
I see I see. Cleaner this way. I had zones and space nested together earlier. I will change it to this format.